.github/workflows/lint.yml

```diff
@@ -6,8 +6,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out code into the Go module directory
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
       - name: golangci-lint
-        uses: reviewdog/action-golangci-lint@v1
+        uses: docker://haproxytech/linter:1.33
         with:
-          github_token: ${{ secrets.github_token }}
```

Comment:
I think it's safe enough removing the `github_token` since there's no more need.